platform/darwin/src/MGLMapSnapshotter.mm

```diff
@@ -550,7 +550,7 @@ - (void)setOptions:(MGLMapSnapshotOptions *)options
     if (CLLocationCoordinate2DIsValid(options.camera.centerCoordinate)) {
         cameraOptions.center = MGLLatLngFromLocationCoordinate2D(options.camera.centerCoordinate);
     }
-    cameraOptions.angle = MAX(0, options.camera.heading) * mbgl::util::DEG2RAD;
+    cameraOptions.angle = MAX(0, options.camera.heading);
```

Comment:
Before this PR, here was a bug where `MGLMapSnapshotOptions.camera`’s `heading` was misinterpreted to be counterclockwise instead of clockwise from north.